libglusterfs/src/locking.c

```diff
@@ -8,7 +8,6 @@
   cases as published by the Free Software Foundation.
 */
 
-#if defined(HAVE_SPINLOCK)
```

Comment:
Without spinlocks this whole file doesn't make sense. There's nothing to compile. It should be removed.